src/nvim/eval.c

```diff
@@ -22796,6 +22886,23 @@ void call_user_func(ufunc_T *fp, int argcount, typval_T *argvars,
       if (islambda) {
         addlocal = true;
       }
+
+      // evaluate named argument default expression
+      isdefault = ai + fp->uf_def_args.ga_len >= 0
+        && (i >= argcount || (argvars[i].v_type == VAR_SPECIAL
+                              && argvars[i].vval.v_number == kSpecialVarNull));
```

Comment:
Since neovim does not have `v:none`, `v:null` is used instead. Should this feature be omitted? It seems difficult to define `v:none` because there is no type corresponding to `msgpack`. Please let me know if you have a good idea.

Reply:
We do not want `v:none` in any form. It is useless, and only exists to provide "loose" JSON in Vim, whereas the Nvim JSON parser only supports valid JSON.

Reply:
This should be omitted entirely. `v:null` is a perfectly valid value (in both vim and nvim), it should not be implicitly replaced by the default.

Reply:
The Vim `v:none` behavior for "default args" will make some scripts break since Nvim doesn't support `v:none`. But I think we should still avoid `v:none`. We're going to have to make some choices as Vim continues to add weird Vimscript features.